packages/trpc/server/routers/viewer/availability/team/listTeamAvailability.handler.ts

```diff
@@ -116,23 +103,15 @@ async function getInfoForAllTeams({ ctx, input }: GetOptions) {
       },
       select: {
         id: true,
+        teamId: true,
       },
     })
-    .then((memberships) => memberships.map((membership) => membership.id));
+    .then((memberships) => memberships.map((membership) => membership.teamId));
 
   if (!teamIds.length) {
     throw new TRPCError({ code: "NOT_FOUND", message: "User is not part of any organization or team." });
   }
 
-  const getTotalMembers = await prisma.$queryRaw<{
-    count: number;
-  }>(Prisma.sql`
-      SELECT
-        COUNT(DISTINCT "userId") as "count"
-      FROM "Membership"
-      WHERE "teamId" IN (${Prisma.join(teamIds)})
-`);
```

Comment:
We actually do the distinct in the membership query so we can rely on that for the correct count value